<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="files_gBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="source_label">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="source_text"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="location_label">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="location_text"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="source_select">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="location_select">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>172</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>681</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="changes_tab">
     <attribute name="title">
      <string>Changes</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2history_tab">
     <attribute name="title">
      <string>History</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="backups_gBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Backups</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="push_button">
        <property name="text">
         <string>Push</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pull_button">
        <property name="text">
         <string>Pull</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboboxBackup"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="currentBackup">
        <property name="text">
         <string>Current Backup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="schedule_label">
       <property name="text">
        <string>Select Schedule</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scheduleBox">
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monthly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Every Save</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="type_label">
       <property name="text">
        <string>Select Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeBox">
       <item>
        <property name="text">
         <string>Folder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Changed Files</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExisting"/>
    <addaction name="actionClone"/>
    <addaction name="actionOptions"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionselectAll"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionChanges"/>
    <addaction name="actionHistory"/>
    <addaction name="actionBackup_List"/>
    <addaction name="actiondevTools"/>
   </widget>
   <widget class="QMenu" name="menuBackups">
    <property name="title">
     <string>Backups</string>
    </property>
    <addaction name="actionPush"/>
    <addaction name="actionPull"/>
    <addaction name="actionFetch"/>
    <addaction name="actionRemove"/>
    <addaction name="actionView_on_Browser"/>
    <addaction name="actionShow_in_Explorer"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReport_Issue"/>
    <addaction name="actionContact_Support"/>
    <addaction name="actionShow_Keyboard_Shortcuts"/>
    <addaction name="actionShow_Logs"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuView"/>
   <addaction name="menuBackups"/>
   <addaction name="menuHistory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExisting">
   <property name="text">
    <string>Existing</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionselectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionChanges">
   <property name="text">
    <string>Changes</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionBackup_List">
   <property name="text">
    <string>Backup List</string>
   </property>
  </action>
  <action name="actiondevTools">
   <property name="text">
    <string>Developer Tools</string>
   </property>
  </action>
  <action name="actionPush">
   <property name="text">
    <string>Push</string>
   </property>
  </action>
  <action name="actionPull">
   <property name="text">
    <string>Pull</string>
   </property>
  </action>
  <action name="actionFetch">
   <property name="text">
    <string>Fetch</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionView_on_Browser">
   <property name="text">
    <string>View on Browser</string>
   </property>
  </action>
  <action name="actionShow_in_Explorer">
   <property name="text">
    <string>Show in Explorer</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionReport_Issue">
   <property name="text">
    <string>Report Issue</string>
   </property>
  </action>
  <action name="actionContact_Support">
   <property name="text">
    <string>Contact Support</string>
   </property>
  </action>
  <action name="actionShow_Keyboard_Shortcuts">
   <property name="text">
    <string>Show Keyboard Shortcuts</string>
   </property>
  </action>
  <action name="actionShow_Logs">
   <property name="text">
    <string>Show Logs</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
